src/Root.cc

```diff
@@ -140,6 +140,40 @@ Root::Root()
 {
 }
 
+Errors Root::WorldName(const std::string &_filename, std::string &_worldName)
```

Comment:
Remove this change.

Reply:
https://github.com/gazebosim/sdformat/pull/997/commits/48698b0d1b7e87b165fdb2bc6e0dbca04d4cba75